packages/contracts/contracts/TellerV2.sol

```diff
@@ -213,6 +222,29 @@ contract TellerV2 is
         collateralManager = ICollateralManager(_collateralManager);
     }
 
+    function setLenderManager(address _lenderManager)
+        public
```

Comment:
since we have an `internal` function as well, we can make this `external` instead of `public` - same for the `setCollateralManager` function above actually